packages/contracts-bedrock/test/CrossDomainOwnable.t.sol

```diff
@@ -2,17 +2,18 @@
 pragma solidity 0.8.15;
 
 // Testing utilities
-import { Vm, VmSafe } from "forge-std/Vm.sol";
+import { VmSafe } from "forge-std/Vm.sol";
+import { Test } from "forge-std/Test.sol";
 import { CommonTest, Portal_Initializer } from "test/CommonTest.t.sol";
```

Comment:
nit: we can now remove the `CommonTest` impor

Reply:
good catch!